src/librustc/util/profiling.rs

```diff
@@ -197,7 +197,21 @@ impl SelfProfiler {
     }
 
     fn stop_timer(&mut self) -> u64 {
-        let elapsed = self.current_timer.elapsed();
+        let elapsed =
+            if cfg!(windows) {
```

Comment:
Can you change the formatting to have the `if` clause in the same line as `let elapsed` and move the rest of the block one indentation level left?